tests/w3c_tracecontext_validation_server.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
```

Comment:
This test should stay in the core repo as well, since this will be testing trace context propagation which is part of the API